draft/spec/index.html

```diff
@@ -953,7 +963,7 @@ <h2>Storage Hierarchies</h2>
     </section>
 
     <section id="root-extensions">
-        <h2>Extensions</h2>
+        <h2>Root Extensions</h2>
```

Comment:
This makes it sound like their is a difference between extensions that appear in the storage root and the object root.  I thought we had agreed that Extensions would be stored at the root and then used in the objects -- but you might not use all extensions listed in the storage root.

Reply:
There are extensions, like the “layout” functionality, that will be root-only. In this case it does not affect the contents of objects, but only the paths to them.

Reply:
Change title to "Storage Root Extensions"

Reply:
... which still leave the question of where descriptions of object extensions go

Reply:
Descriptions of object extensions also go in the storage root as a plain text file, on the theory that it is a single place that can describe the structure of some or all objects in that storage root.

Reply:
Agree with @ahankinson